book/validation.rst

```diff
@@ -511,8 +511,8 @@ For information on translating the constraint messages, see
 Constraint Targets
 ------------------
 
-Constraints can be applied to a class property (e.g. ``name``) or a public
-getter method (e.g. ``getFullName``). The first is the most common and easy
+Constraints can be applied to a class property (e.g. ``name``), a public
+getter method (e.g. ``getFullName``) or an entire class. The first is the most common and easy
```

Comment:
In the following sentence we mention "the first" and "the second" which is not exact anymore as we now have three things (property, method, and class).